src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -99,10 +101,14 @@ public void toggleConeMode() {
     isConeMode = !isConeMode;
   }
 
+  public void setCubeOverride(boolean cubeOverride){
+    this.cubeOverride = cubeOverride;
+  }
+
   public boolean hasCube() {
     int cubeDistance = timeOfFlightSensor.getDistance1();
     boolean cubeSensorUp = cubeDistance != -1;
-    return cubeDistance <= mmCubeActivationThreshold && cubeSensorUp;
+    return cubeOverride || (cubeDistance <= mmCubeActivationThreshold && cubeSensorUp);
```

Comment:
Reporting this value in Shuffleboard like we do for "Intake Mode" would be good for the Operator to easily see when they need to hit the override button